src/edge/importer.py

```diff
@@ -107,9 +110,30 @@ def build_fragment(self):
         prev = None
         flen = 0
         seqlen = len(self.__sequence)
+
+	# divide chunks bigger than a certain threshold to smaller chunks, to
+	# allow insertion of sequence into database. e.g. MySQL has a packet
+	# size that prevents chunks that are too large from being inserted.
+        cs_limit = 1000000
```

Comment:
Thanks for updating the variable names, super clear now. Do you mind just change this cs_limit to chunk_size_limit to be consistent? Thanks!